src/relax/op/tensor/manipulate.cc

```diff
@@ -595,6 +597,8 @@ TVM_REGISTER_OP("relax.permute_dims")
     .set_num_inputs(1)
     .add_argument("x", "Tensor", "The input tensor.")
     .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoPermuteDims)
+    .set_attr<FInferStructInfo>("dist.FInferStructInfo",
```

Comment:
You can directly register in distributed/manipulate.cc

it is possible to register in different places 

```
TVM_REGISTER_OP("relax.permute_dims")
.set_attr<FInferStructInfo>("dist.FInferStructInfo", distributed::InferDistStructInfoReshape)
```